bittensor/constants.py

```diff
@@ -15,7 +15,29 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
+# Standard Library
+import asyncio
+from typing import Dict, Type
+
+# 3rd Party
+import aiohttp
+
 
 ALLOWED_DELTA = 4_000_000_000  # Delta of 4 seconds for nonce validation
 V_7_2_0 = 7002000
 NANOSECONDS_IN_SECOND = 1_000_000_000
+
+#### Dendrite ####
+DENDRITE_ERROR_MAPPING: Dict[Type[Exception], tuple] = {
+    aiohttp.ClientConnectorError: ("503", "Service unavailable"),
+    asyncio.TimeoutError: ("408", "Request timeout"),
+    aiohttp.ClientResponseError: (None, "Client response error"),
+    aiohttp.ClientPayloadError: ("400", "Payload error"),
+    aiohttp.ClientError: ("500", "Client error"),
+    aiohttp.ServerTimeoutError: ("504", "Server timeout error"),
+    aiohttp.ServerDisconnectedError: ("503", "Service disconnected"),
+    aiohttp.ServerConnectionError: ("503", "Service connection error"),
```

Comment:
I'm totally fine with the hard-coding here, but just for future reference, the error message is embedded in the docstring, so like `aiohttp.ServerTimeoutError.__doc__`.